api/src/test/java/org/openmrs/module/openconceptlab/updater/ImporterTest.java

```diff
@@ -464,6 +468,47 @@ public void importConcept_shouldChangeDuplicateSynonymToIndexTerm() throws Excep
 		assertThat(importedConceptWithIndexTerm.getNames(), hasItem((Matcher<? super ConceptName>) allOf(hasProperty("conceptNameType", equalTo(ConceptNameType.INDEX_TERM)),
 			hasProperty("name", equalTo("Nazwa")))));
 	}
+	
+	/**
+	 * @see Importer#importConcept(OclConcept,ImportQueue)
+	 * @verifies save concept if versionUrl changed from last update
+	 */
+	@Test
+	public void importConcept_shouldUpdateConceptIfVersionUrlChanged() throws Exception {

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method names should not contain underscores](https://www.codacy.com/app/agrzybkowski/openmrs-module-openconceptlab/file/2377174879/issues/source?bid=3218637&fileBranchId=3224807#l477)